### PR TITLE
docs: add a notice prohibiting retention policies on backup targets

### PR DESCRIPTION
**Pull request by derekbit**
#### Which issue(s) this PR fixes:

Issue Longhorn/longhorn#9795

### CODE REVIEW

**Review by innobead — APPROVED** [review]
Technically LGTM.

File: content/docs/1.6.0/snapshots-and-backups/backup-and-restore/set-backup-target.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ If you don't have access to AWS S3 or want to give the backupstore a try first,
 
 Longhorn also supports setting up recurring snapshot/backup jobs for volumes, via Longhorn UI or Kubernetes Storage Class. See [here](../../scheduling-backups-and-snapshots) for details.
 
+> **Notice**
+> 
+> The lifecycle of Longhorn backups within the backup store is entirely managed by Longhorn. Any retention policy directly on the backup store is strictly prohibited.
```
Comment (jhkrug) [inline content/docs/1.6.0/snapshots-and-backups/backup-and-restore/set-backup-target.md:18]:
```suggestion
> The lifecycle of Longhorn backups within the backup store is managed by Longhorn. Any retention policy on the backup store is prohibited.
```

File: content/docs/1.6.1/snapshots-and-backups/backup-and-restore/set-backup-target.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ If you don't have access to AWS S3 or want to give the backupstore a try first,
 
 Longhorn also supports setting up recurring snapshot/backup jobs for volumes, via Longhorn UI or Kubernetes Storage Class. See [here](../../scheduling-backups-and-snapshots) for details.
 
+> **Notice**
+> 
+> The lifecycle of Longhorn backups within the backup store is entirely managed by Longhorn. Any retention policy directly on the backup store is strictly prohibited.
```
Comment (jhkrug) [inline content/docs/1.6.1/snapshots-and-backups/backup-and-restore/set-backup-target.md:18]:
```suggestion
> The lifecycle of Longhorn backups within the backup store is managed by Longhorn. Any retention policy on the backup store is prohibited.
```

File: content/docs/1.6.2/snapshots-and-backups/backup-and-restore/set-backup-target.md [hunk 3]
```diff
@@ -13,6 +13,10 @@ If you don't have access to AWS S3 or want to give the backupstore a try first,
 
 Longhorn also supports setting up recurring snapshot/backup jobs for volumes, via Longhorn UI or Kubernetes Storage Class. See [here](../../scheduling-backups-and-snapshots) for details.
 
+> **Notice**
+> 
+> The lifecycle of Longhorn backups within the backup store is entirely managed by Longhorn. Any retention policy directly on the backup store is strictly prohibited.
```
Comment (jhkrug) [inline content/docs/1.6.2/snapshots-and-backups/backup-and-restore/set-backup-target.md:18]:
```suggestion
> The lifecycle of Longhorn backups within the backup store is managed by Longhorn. Any retention policy on the backup store is prohibited.
```

File: content/docs/1.6.3/snapshots-and-backups/backup-and-restore/set-backup-target.md [hunk 4]
```diff
@@ -13,6 +13,10 @@ If you don't have access to AWS S3 or want to give the backupstore a try first,
 
 Longhorn also supports setting up recurring snapshot/backup jobs for volumes, via Longhorn UI or Kubernetes Storage Class. See [here](../../scheduling-backups-and-snapshots) for details.
 
+> **Notice**
+> 
+> The lifecycle of Longhorn backups within the backup store is entirely managed by Longhorn. Any retention policy directly on the backup store is strictly prohibited.
```
Comment (jhkrug) [inline content/docs/1.6.3/snapshots-and-backups/backup-and-restore/set-backup-target.md:18]:
```suggestion
> The lifecycle of Longhorn backups within the backup store is managed by Longhorn. Any retention policy on the backup store is prohibited.
```

File: content/docs/1.6.4/snapshots-and-backups/backup-and-restore/set-backup-target.md [hunk 5]
```diff
@@ -13,13 +13,17 @@ If you don't have access to AWS S3 or want to give the backupstore a try first,
 
 Longhorn also supports setting up recurring snapshot/backup jobs for volumes, via Longhorn UI or Kubernetes Storage Class. See [here](../../scheduling-backups-and-snapshots) for details.
 
-> Note:  
-> Longhorn will attempt to clean up the backup-related custom resources in the following scenarios:
+> **Notice**
+> 
+> - The lifecycle of Longhorn backups within the backup store is entirely managed by Longhorn. **Any retention policy directly on the backup store is strictly prohibited**.
```
Comment (jhkrug) [inline content/docs/1.6.4/snapshots-and-backups/backup-and-restore/set-backup-target.md:18]:
```suggestion
> - The lifecycle of Longhorn backups within the backup store is managed by Longhorn. **Any retention policy on the backup store is prohibited**.
```

File: content/docs/1.6.4/snapshots-and-backups/backup-and-restore/set-backup-target.md [hunk 6]
```diff
@@ -13,13 +13,17 @@ If you don't have access to AWS S3 or want to give the backupstore a try first,
...
 >
-> - An empty response from the NFS server due to a brief server downtime.
-> - A race condition between related Longhorn backup controllers.
->
-> The backup information will be resynchronized during the next polling interval. For more information, see [#9530](https://github.com/longhorn/longhorn/issues/9530).
+> - Longhorn will attempt to clean up the backup-related custom resources in the following scenarios:
```
Comment (jhkrug) [inline content/docs/1.6.4/snapshots-and-backups/backup-and-restore/set-backup-target.md:20]:
```suggestion
> - Longhorn attempts to clean up the backup-related custom resources in the following scenarios:
```

File: content/docs/1.6.4/snapshots-and-backups/backup-and-restore/set-backup-target.md [hunk 7]
```diff
@@ -13,13 +13,17 @@ If you don't have access to AWS S3 or want to give the backupstore a try first,
...
+>   - An empty response from the NFS server due to a brief server downtime.
```
Comment (jhkrug) [inline content/docs/1.6.4/snapshots-and-backups/backup-and-restore/set-backup-target.md:21]: Is the brevity of the server downtime relevant here?
```suggestion
>   - An empty response from the NFS server due to server downtime.
```

File: content/docs/1.7.0/snapshots-and-backups/backup-and-restore/set-backup-target.md [hunk 8]
```diff
@@ -13,6 +13,10 @@ If you don't have access to AWS S3 or want to give the backupstore a try first,
 
 Longhorn also supports setting up recurring snapshot/backup jobs for volumes, via Longhorn UI or Kubernetes Storage Class. See [here](../../scheduling-backups-and-snapshots) for details.
 
+> **Notice**
+> 
+> The lifecycle of Longhorn backups within the backup store is entirely managed by Longhorn. Any retention policy directly on the backup store is strictly prohibited.
```
Comment (jhkrug) [inline content/docs/1.7.0/snapshots-and-backups/backup-and-restore/set-backup-target.md:18]:
```suggestion
> The lifecycle of Longhorn backups within the backup store is managed by Longhorn. Any retention policy on the backup store is prohibited.
```

File: content/docs/1.7.1/snapshots-and-backups/backup-and-restore/set-backup-target.md [hunk 9]
```diff
@@ -13,6 +13,10 @@ If you don't have access to AWS S3 or want to give the backupstore a try first,
 
 Longhorn also supports setting up recurring snapshot/backup jobs for volumes, via Longhorn UI or Kubernetes Storage Class. See [here](../../scheduling-backups-and-snapshots) for details.
 
+> **Notice**
+> 
+> The lifecycle of Longhorn backups within the backup store is entirely managed by Longhorn. Any retention policy directly on the backup store is strictly prohibited.
```
Comment (jhkrug) [inline content/docs/1.7.1/snapshots-and-backups/backup-and-restore/set-backup-target.md:18]:
```suggestion
> The lifecycle of Longhorn backups within the backup store is managed by Longhorn. Any retention policy on the backup store is prohibited.
```

File: content/docs/1.7.2/snapshots-and-backups/backup-and-restore/set-backup-target.md [hunk 10]
```diff
@@ -13,13 +13,15 @@ If you don't have access to AWS S3 or want to give the backupstore a try first,
 
 Longhorn also supports setting up recurring snapshot/backup jobs for volumes, via Longhorn UI or Kubernetes Storage Class. See [here](../../scheduling-backups-and-snapshots) for details.
 
-> Note:  
-> Longhorn will attempt to clean up the backup-related custom resources in the following scenarios:
+> **Notice**
+> 
+> - The lifecycle of Longhorn backups within the backup store is entirely managed by Longhorn. **Any retention policy directly on the backup store is strictly prohibited**.
```
Comment (jhkrug) [inline content/docs/1.7.2/snapshots-and-backups/backup-and-restore/set-backup-target.md:18]:
```suggestion
> - The lifecycle of Longhorn backups within the backup store is managed by Longhorn. **Any retention policy on the backup store is prohibited**.
```

File: content/docs/1.7.2/snapshots-and-backups/backup-and-restore/set-backup-target.md [hunk 11]
```diff
@@ -13,13 +13,15 @@ If you don't have access to AWS S3 or want to give the backupstore a try first,
...
 >
-> - An empty response from the NFS server due to a brief server downtime.
-> - A race condition between related Longhorn backup controllers.
->
-> The backup information will be resynchronized during the next polling interval. For more information, see [#9530](https://github.com/longhorn/longhorn/issues/9530).
+> - Longhorn will attempt to clean up the backup-related custom resources in the following scenarios:
```
Comment (jhkrug) [inline content/docs/1.7.2/snapshots-and-backups/backup-and-restore/set-backup-target.md:20]:
```suggestion
> - Longhorn attempts to clean up the backup-related custom resources in the following scenarios:
```

File: content/docs/1.7.2/snapshots-and-backups/backup-and-restore/set-backup-target.md [hunk 12]
```diff
@@ -13,13 +13,15 @@ If you don't have access to AWS S3 or want to give the backupstore a try first,
...
+>   - An empty response from the NFS server due to a brief server downtime.
```
Comment (jhkrug) [inline content/docs/1.7.2/snapshots-and-backups/backup-and-restore/set-backup-target.md:21]:
```suggestion
>   - An empty response from the NFS server due to server downtime.
```

File: content/docs/1.7.3/snapshots-and-backups/backup-and-restore/set-backup-target.md [hunk 13]
```diff
@@ -13,13 +13,15 @@ If you don't have access to AWS S3 or want to give the backupstore a try first,
 
 Longhorn also supports setting up recurring snapshot/backup jobs for volumes, via Longhorn UI or Kubernetes Storage Class. See [here](../../scheduling-backups-and-snapshots) for details.
 
-> Note:  
-> Longhorn will attempt to clean up the backup-related custom resources in the following scenarios:
+> **Notice**
+> 
+> - The lifecycle of Longhorn backups within the backup store is entirely managed by Longhorn. **Any retention policy directly on the backup store is strictly prohibited**.
```
Comment (jhkrug) [inline content/docs/1.7.3/snapshots-and-backups/backup-and-restore/set-backup-target.md:18]:
```suggestion
> - The lifecycle of Longhorn backups within the backup store is managed by Longhorn. **Any retention policy on the backup store is prohibited**.
```

File: content/docs/1.7.3/snapshots-and-backups/backup-and-restore/set-backup-target.md [hunk 14]
```diff
@@ -13,13 +13,15 @@ If you don't have access to AWS S3 or want to give the backupstore a try first,
...
 >
-> - An empty response from the NFS server due to a brief server downtime.
-> - A race condition between related Longhorn backup controllers.
->
-> The backup information will be resynchronized during the next polling interval. For more information, see [#9530](https://github.com/longhorn/longhorn/issues/9530).
+> - Longhorn will attempt to clean up the backup-related custom resources in the following scenarios:
```
Comment (jhkrug) [inline content/docs/1.7.3/snapshots-and-backups/backup-and-restore/set-backup-target.md:20]:
```suggestion
> - Longhorn attempts to clean up the backup-related custom resources in the following scenarios:
```

File: content/docs/1.7.3/snapshots-and-backups/backup-and-restore/set-backup-target.md [hunk 15]
```diff
@@ -13,13 +13,15 @@ If you don't have access to AWS S3 or want to give the backupstore a try first,
...
+>   - An empty response from the NFS server due to a brief server downtime.
```
Comment (jhkrug) [inline content/docs/1.7.3/snapshots-and-backups/backup-and-restore/set-backup-target.md:21]:
```suggestion
>   - An empty response from the NFS server due to server downtime.
```

File: content/docs/1.7.3/snapshots-and-backups/backup-and-restore/set-backup-target.md [hunk 16]
```diff
@@ -13,13 +13,15 @@ If you don't have access to AWS S3 or want to give the backupstore a try first,
...
+>   - A race condition between related Longhorn backup controllers.
+> 
+>   The backup information will be resynchronized during the next polling interval. For more information, see [#9530](https://github.com/longhorn/longhorn/issues/9530).
```
Comment (jhkrug) [inline content/docs/1.7.3/snapshots-and-backups/backup-and-restore/set-backup-target.md:24]:
```suggestion
>   The backup information is resynchronized during the next polling interval. For more information, see [#9530](https://github.com/longhorn/longhorn/issues/9530).
```

File: content/docs/1.8.0/snapshots-and-backups/backup-and-restore/set-backup-target.md [hunk 17]
```diff
@@ -13,13 +13,15 @@ If you don't have access to AWS S3 or want to give the backupstore a try first,
 
 Longhorn also supports setting up recurring snapshot/backup jobs for volumes, via Longhorn UI or Kubernetes Storage Class. See [here](../../scheduling-backups-and-snapshots) for details.
 
-> Note:  
-> Longhorn will attempt to clean up the backup-related custom resources in the following scenarios:
+> **Notice**
+> 
+> - The lifecycle of Longhorn backups within the backup store is entirely managed by Longhorn. **Any retention policy directly on the backup store is strictly prohibited**.
```
Comment (jhkrug) [inline content/docs/1.8.0/snapshots-and-backups/backup-and-restore/set-backup-target.md:18]:
```suggestion
> - The lifecycle of Longhorn backups within the backup store is managed by Longhorn. **Any retention policy on the backup store is prohibited**.
```

**Review by jhkrug — CHANGES_REQUESTED** [review]
Some minor suggestions for your consideration. Thanks.

**Review by jhkrug — APPROVED** [review]
lgtm.

**Review by innobead — APPROVED** [review]

**Pull request merged**